packages/frontend/components/ui/avatar.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cls from 'classnames';
+
+function Avatar({ type, className, image }) {
+  const avatarType = {
+    xs: 'w-8 h-8',
+    normal: 'w-11 h-11',
+    lg: 'w-14 h-14',
+  };
+
+  return (
+    <div
+      className={cls(
+        avatarType[type],
+        'flex align-center justify-center overflow-hidden',
+        className
+      )}
+    >
+      {image ? (
+        <img
+          className="bg-no-repeat bg-center bg-contain rounded-full"
```

Comment:
are you sure about using `rounded-full` in here? i guess you can give `rounded-full` to parent div. img isn't the rounded one, container is. can you check this?